docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
         env_file:
             - docker-compose.env.yaml
         environment:
-            DATABASE_URL: postgis://ahti:ahti@ahti-db/ahti
+            DATABASE_URL: postgres://ahti:ahti@ahti-db/ahti
```

Comment:
ahti not using gis anymore?

Reply:
It is.. it's enforced in `settings.py`. `psql` command understands only db addresses that start with `postgres://`. AFAIK the DATABASE_URL configured ahti in the CI/CD pipeline both begin with `postgres`.

Reply:
Basically I wanted to avoid adding a second setting for a database url with this particular change.

Reply:
ah, okay 🙂